airo-camera-toolkit/airo_camera_toolkit/calibration/README.md

```diff
@@ -0,0 +1,92 @@
+# Hand-eye calibration
+
+The `calibration` submodule of the `airo-camera-toolkit` contains the code for hand-eye calibration (also called extrinsics calibration).
+If you already know what this is, you can skip to the [quick start](#quick-start) section.
+
+The goal of hand-eye calibration is to find the transformation between the camera and the robot.
+There are two variants, depending on how the camera is mounted:
+
+- `eye_in_hand`: the camera is mount on the wrist of the robot, the transform that we want is from TCP to camera.
+- `eye_to_hand`: the camera is mounted on a fixed position in the robot's workspace, the transform that we want is from robot base to camera.
+
+To compute this transform, we use a Charuco board, which's pose relative to the camera can be accurately detected from a single image.
+In `eye_in_hand` mode the board is fixed in the workspace of the robot and we move the wrist camera around.
+In `eye_to_hand` mode the board is grasped by the robot and moved around, and the camera is stationary.
+Then using at least 3 samples of the form: `(tcp_pose_in_robot_base_frame, board_pose_in_camera_frame)` we can solve a non-linear system of equations to find the camera's pose.
+
+## Quick start
+
+**Starting the calibration**:
+
+1. Place or grasp the Charuco board firmly (grasp pose can be chosen freely)
+2. Put the robot in `Remote Control` mode
```

Comment:
This is UR specific, but I guess so is the calibration for now (with the freedrive?)

Reply:
Indeed it's mostly the freedrive part that is making the code UR-specific. We'll have to generalize that when we get a new robot.